### PR TITLE
Update my bio

### PR DESCRIPTION
**Pull request by shivam-sharma7**
Update my bio

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1321"></a>

### CODE REVIEW

File: public/data/shivam-sharma7.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Shivam Sharma",
   "type": "personal",
-  "bio": "Member of @EddieHubCommunity, @fossasia | Full Stack Web Developer | Open Source & Cloud & DevOps Enthusiast |",
+  "bio": "Member of @EddieHubCommunity | Full Stack Web Developer | Open Source & Cloud & DevOps Enthusiast |",
```
Comment (vikasganiga05) [inline public/data/shivam-sharma7.json:4]:
```suggestion
  "bio": "Member of @EddieHubCommunity | Full Stack Web Developer | Open Source & Cloud & DevOps Enthusiast.",
```
Replace the vertical bar(`|`) with a full stop(`.`)

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
Your bio is ending with a vertical bar(`|`) character. I think you should change end it with a full stop(`.`) character.

File: public/data/shivam-sharma7.json (hunk 1 above)
Comment (shivam-sharma7) [reply public/data/shivam-sharma7.json]: Okay @vikasganiga05

**Review by schmelto — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by vikasganiga05 — APPROVED** [review]
Now it looks good to me. Thanks for updating your profile :pray:

**Comment by shivam-sharma7** [comment]
Seem good to me then I did

**Review by kumarsonsoff3 — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**